pkg/genlib/generator_interface.go

```diff
@@ -567,7 +567,48 @@ func fuzzyInt(previous int64, fuzziness, min, max float64) int64 {
 	return customRand.Int63n(int64(math.Ceil(higherBound-lowerBound))) + int64(lowerBound)
 }
 
+func fuzzyIntCounter(previous int64, fuzziness float64) int64 {
+	lowerBound := float64(previous)
+	higherBound := float64(previous) * (1 + fuzziness)
```

Comment:
fuzziness is to have larger increments?

Reply:
fuzziness allows to specify the maximum delta a generated value can have from the previous value.
If you have an int and you don't have fuzziness, you can have a series of generated value like: "12, 1765, 123, 87610"

The same if you have a range. Imagine a cpu usage field, 0-100: "2, 98, 37, 12, 78"

If you don't want those "jumps", but rather something smoother, you rely on fuzziness.

For the cpu usage field a series with 20% delta is probably more realistic:
```
- name: cpu
  range:
    min: 0
    max: 100
  fuzziness: 0.2
```

the series will look more like: "10, 12, 9, 7"